index.html

```diff
@@ -41,6 +47,103 @@ <h3 class="supporting-text">Software Engineering</h3>
+          <p>Selenium</p>
+        </div>
+
+        <div class="skills">
+          <h2>Skills</h2>
+          <p>Codekit</p>
+          <p>GitHub</p>
+          <p>Codepen</p>
+          <p>Gitlab</p>
+          <p>Terminal</p>
+        </div>
+        <div class="vector5"></div>
+      </section>
+
+      <div class="icons">
+        <i class="fa-brands fa-github"></i>
+        <i class="fab fa-linkedin-in"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <i class="fab fa-medium"></i>
+      </div>
+
+      <section class="my-work">
+        <div class="work-tag"><i class="fa-thin fa-arrow-down"></i>Work</div>
+
+        <h1>MY RECENT WORK</h1>
+
+        <div class="vector6"></div>
+
+        <div class="my-work-1">
+          <div class="img-placeholder"></div>
+          <h2 class="work-title">Multi-Post Stories</h2>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <h2 class="tech-used">css | html | bootstrap | Ruby</h2>
+
+          <button class="button" type="button">Button</button>
+        </div>
+
+        <div class="my-work-2">
+          <div class="img-placeholder"></div>
+          <h2 class="work-title">Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <div class="tech-used">css | html | bootstrap | Ruby</div>
+
+          <button class="button" type="button">Button</button>
+        </div>
+
+        <div class="my-work-3">
+          <div class="img-placeholder"></div>
+          <h2 class="work-title">Multi-Post Stories</h2>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <div class="tech-used">css | html | bootstrap | Ruby</div>
+          <button class="button" type="button">Button</button>
+        </div>
+
+        <div class="my-work-4">
+          <div class="img-placeholder"></div>
+          <h2 class="work-title">Multi-Post Stories</h2>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <div class="tech-used">css | html | bootstrap | Ruby</div>
+          <button class="button" type="button">Button</button>
+        </div>
+      </section>
     </header>
```

Comment:
- [ ] Please remove your `header` closing tag from this section, this is not supposed to be the main container of your page, the header should contain only the logo and the hamburger.